my-app/src/services/departments.service.js

```diff
@@ -0,0 +1,32 @@
+import Constants from "../constants/constants.service";
+import { getToken } from "./auth.service";
+import { Component } from "react";
+
+const baseUrl = Constants.getBaseUrl();
+const options = {
+  method: "GET",
+  headers: {
+    authorization: getToken(),
+  },
+};
+
+export default class DepartmentsService extends Component {
```

Comment:
Why do you extends from Component here?

Reply:
I really thought I should always be using extends Component